rcl/test/rcl/test_subscription.cpp

```diff
@@ -129,7 +129,7 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
   //  Process     : test_subscription__rmw_opensplice_cpp <23524>
   //  Thread      : main thread 7fff7342d000
   //  Internals   : V6.4.140407OSS///v_topicNew/v_topic.c/448/21/1455157023.781423000
-  const char * topic = "chatter_int64";
+  const char * topic = "rcl_test_subscription_nominal_chatter_int64";
```

Comment:
I changed the topic name to be more specific to avoid cross-talk. Most tests in the test_rclcpp package also follow this convention.